docs: fix grammer for gateway.md (#4691)
hyperledger/fabric: docs/source/gateway.md

## How the gateway endorses your transaction proposal

In order for a transaction to be successfully committed to the ledger, a sufficient number of endorsements are required in order to satisfy

> the [endorsement policy](endorsement-policies.html). Getting an endorsement from an organization involves connecting to one of its peers and having them simulate (execute) the transaction proposal against their copy of the ledger. The peer simulates the transaction by invoking the chaincode function, as specified by its name and arguments in the proposal, and building (and signing) a read-write set. The read-write set contains the current ledger state and proposed changes in response to the state get/set instructions in that function.

Reviewer: ```suggestion
the [endorsement policy](endorsement-policies.html). Getting an endorsement from an organization involves connecting to one of its peers and having it simulate (execute) the transaction proposal against its copy of the ledger. The peer simulates the transaction by invoking the chaincode function, as specified by its name and arguments in the proposal, and building (and signing) a read-write set. The read-write set contains the current ledger state and proposed changes in response to the state get/set instructions in that function.
```
There's no need to change "it" and "its" to "them" and "their" in the passage. "It" and "its" correctly refer to a singular noun, in this case, "one of its peers", meaning one peer of the organization.
